src/assets/closeIcon.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="13" height="13" viewBox="0 0 13 13" fill="none" xmlns="http://www.w3.org/2000/svg">
+<rect y="11.3137" width="16" height="2" rx="1" transform="rotate(-45 0 11.3137)" fill="#F6F6F6"/>
+<rect x="1.41418" width="16" height="2" rx="1" transform="rotate(45 1.41418 0)" fill="#F6F6F6"/>
+</svg>
```

Comment:
This is nit-picky, but can you rename this file to `close-icon.svg` to follow our file naming pattern? Same thing for `useMediaQuery.js` ==> `use-media-query.js`